client/jetpack-connect/jetpack-new-site/style.scss

```diff
@@ -140,24 +140,8 @@
 	.button {
 		width: 100%;
 		margin-top: 16px;
-		background: $green-jetpack;
-		border-color: darken( $green-jetpack, 15% );
-
-		&:hover,
-		&:focus {
-			border-color: darken( $green-jetpack, 30% );
-		}
-
-		&:focus {
-			box-shadow: 0 0 0 2px lighten( $green-jetpack, 25% );
-		}
 
-		&[disabled],
-		&:disabled {
-			background: tint( $green-jetpack, 50% );
-			border-color: tint( $green-jetpack, 35% );
-			color: $white;
-		}
+		@extend .jetpack-connect__main.jetpack-branded .button.is-primary;
```

Comment:
Weird stuff, SASS can't compile this one and I'm getting a "can't extend nested selectors" error:

![](https://cldup.com/xmuNLShdnd.png)

Reply:
Let's not use `@extend` at all in Calypso. It's technically against guidelines and it's not efficient for a few reasons.

Reply:
👍 I removed the `@extend` in 0aa36092789a4c2f3de7c3b7bc85b6fde4d6d2aa

This builds correctly now. I don't know how I missed that 😕